lib/maths/CTimeSeriesDecompositionDetail.cc

```diff
@@ -355,6 +357,8 @@ bool upgradeTrendModelToVersion6p3(const core_t::TTime bucketLength,
     return true;
 }
 
+const TSizeSizeMap SC_STATES_UPGRADING_TO_VERSION_6_3{{0, 0}, {1, 1}, {2, 1}, {3, 2}, {4, 3}};
```

Comment:
Might be worth adding a comment to say the intention is to map states from `{"INITIAL", "SMALL_TEST", "REGULAR_TEST", "NOT_TESTING", "ERROR"}` to the best equivalent in `{"INITIAL", "TEST", "NOT_TESTING", "ERROR"}`.

Reply:
Yes, good point. I'll add a comment.